.github/workflows/diff.yml

```diff
@@ -0,0 +1,118 @@
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  edit_file:
+    runs-on: ubuntu-latest
+    container: obolibrary/odklite:v1.2.32
+    steps:
+      - uses: actions/checkout@v2
+      # Checks-out main branch under "main" directory
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+          path: master
+      - name: Diff classification
+        run: export ROBOT_JAVA_ARGS=-Xmx6G; robot diff --left master/src/ontology/cl-edit.owl --left-catalog master/src/ontology/catalog-v001.xml --right src/ontology/cl-edit.owl --right-catalog src/ontology/catalog-v001.xml -f markdown -o edit-diff.md
+      - name: Upload diff
+        uses: actions/upload-artifact@v2
+        with:
+          name: edit-diff.md
+          path: edit-diff.md
+  classify_pr:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.2.32
+    steps:
+      - uses: actions/checkout@v2
+      - name: Classify ontology
+        run: cd src/ontology; make IMP=FALSE PAT=FALSE MIR=FALSE cl.owl
+      - name: Upload PR cl.owl
+        uses: actions/upload-artifact@v2
+        with:
+          name: cl-pr.owl
+          path: src/ontology/cl.owl
+          retention-days: 1
+  classify_master:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.2.32
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: Classify ontology
+        run: cd src/ontology; make IMP=FALSE PAT=FALSE MIR=FALSE cl.owl
+      - name: Upload master cl.owl
+        uses: actions/upload-artifact@v2
+        with:
+          name: cl-master.owl
+          path: src/ontology/cl.owl
+          retention-days: 1
+  diff_classification:
+    needs:
+      - classify_pr
+      - classify_master
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.2.32
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download master classification
+        uses: actions/download-artifact@v2
+        with:
+          name: cl-master.owl
+          path: src/ontology/cl-master.owl
+      - name: Download PR classification
+        uses: actions/download-artifact@v2
+        with:
+          name: cl-pr.owl
+          path: src/ontology/cl-pr.owl
+      - name: Diff classification
+        run: export ROBOT_JAVA_ARGS=-Xmx6G; cd src/ontology; robot diff --left cl-master.owl/cl.owl --left-catalog catalog-v001.xml --right cl-pr.owl/cl.owl --right-catalog catalog-v001.xml -f markdown -o classification-diff.md
+      - name: Upload diff
+        uses: actions/upload-artifact@v2
+        with:
+          name: classification-diff.md
+          path: src/ontology/classification-diff.md
+  post_comment:
+    needs: [diff_classification, edit_file]
+    runs-on: ubuntu-latest
+    container: docker://dvcorg/cml-py3:latest
```

Comment:
Why is this needed? Out of curiosity.

Reply:
oh whoops this isn't, was testing with something and totally forgot to remove it. @ubyndr removed in here the last time, but I forgot to remove it on my test fork and copied and paste the whole thing - will remove